server/src/app.service.ts

```diff
@@ -39,24 +39,23 @@ export class GameService {
 
   gamesList(all: boolean): Game[] {
     let result: Game[] = [];
-    if(all) {
-      for(let key in this.games) { 
-        result.push(this.games[key])
-      }
-      return result
-    }
+
     for(let key in this.games) {
       const playar = this.games[key]
-      if(!playar.players.x || !playar.players.o) {
+      if(all) {
```

Comment:
if (all || (!playar.players.x || !playar.players.o)) {
        result.push(this.games[key])
      }